<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>467</width>
     <height>830</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/back/4.배경.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../page4resource.qrc">:/back/4.배경.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="trainer_btn">
   <property name="geometry">
    <rect>
     <x>-18</x>
     <y>433</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button/4.트레이너설명듣기버튼.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="weight_btn">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>428</y>
     <width>209</width>
     <height>49</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button/4.중량설정버튼.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>753</y>
     <width>33</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button/뒤로가기 그룹.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="home_btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>741</y>
     <width>48</width>
     <height>58</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button/홈버튼 그룹.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tip_btn">
   <property name="geometry">
    <rect>
     <x>387</x>
     <y>750</y>
     <width>44</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/button/팁버튼.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../page4resource.qrc"/>
 </resources>
 <connections/>
</ui>
